search-app/src/cmr/search/site/routes.clj

```diff
@@ -2,19 +2,31 @@
   "This namespace is the one responsible for the routes intended for human
   consumption."
   (:require
-   ;; Third-party libs
-   [compojure.core :refer :all]
-   [ring.swagger.ui :as ring-swagger-ui]
+    ;; Third-party libs
+    [compojure.core :refer :all]
+    [ring.swagger.ui :as ring-swagger-ui]
 
-   ;; CMR libs
-   [cmr.collection-renderer.api.routes :as collection-renderer-routes]
-   [cmr.common.api.context :as context]
-   [cmr.common-app.api-docs :as api-docs]))
+    ;; CMR libs
+    [cmr.collection-renderer.api.routes :as collection-renderer-routes]
+    [cmr.common.api.context :as context]
+    [cmr.common-app.api-docs :as api-docs]
+    [cmr.search.site.pages :as pages]))
 
 (defn build-routes [system]
   (let [relative-root-url (get-in system [:public-conf :relative-root-url])]
     (routes
       (context relative-root-url []
+        ;; Landing pages - Note that the landing pages must come before the
+        ;; api-docs, since api-docs/docs-routes also use a context of "site"
+        ;; but have the last entry as a 404 renderer, and as such, would
+        ;; prevent any pages in the "site" context from rendering after that
+        ;; point.
+        (GET "/" request
+          (pages/home request))
+        (GET "/site/collections/landing-pages" request
```

Comment:
Okay, added site/non-api docs. That actually took longer than expected ... mostly because I started to make a bunch of changes to how docs are done, and then backed those out ;-) 

(Maybe we can discuss that in the future, and if it seems worth it, have a backlog ticket for improving how docs are managed and accessed for CMR apps ...)